Modules/Filtering/ImageCompare/test/itkSTAPLEImageFilterTest.cxx

```diff
@@ -158,10 +158,10 @@ Stapler<VDimension>::Execute()
   typename itk::ImageFileReader<InputImageType>::Pointer  reader;
   typename itk::ImageFileWriter<OutputImageType>::Pointer writer = itk::ImageFileWriter<OutputImageType>::New();
 
-  size_t number_of_files = m_Files.size();
+  size_t numberOfFiles = m_Files.size();
```

Comment:
Commit message body redundant with summary.

Reply:
I always put a commit body; event for commits where the subject line is sufficiently descriptive.